be/test/storage/tablet_updates_test.cpp

```diff
@@ -317,6 +357,35 @@ static ssize_t read_and_compare(const vectorized::ChunkIteratorPtr& iter, const
     return count;
 }
 
+static ssize_t read_and_compare_schema_changed(const vectorized::ChunkIteratorPtr& iter, const vector<int64_t>& keys) {
+    auto chunk = vectorized::ChunkHelper::new_chunk(iter->schema(), 100);
```

Comment:
Why not pass a chunk instead of keys?
If you use keys as a parameter, the different case may need different chunk. 
You have different construct function from keys.

Reply:
yep, changed.